docs/root/source/transaction-concepts.md

```diff
@@ -12,11 +12,18 @@ A creation transaction also establishes the conditions that must be met to trans
 
 A _transfer transaction_ can transfer an asset by fulfilling the current conditions on the asset. It can also specify new transfer conditions.
 
-Today, every transaction contains one fulfillment-condition pair. The fulfillment in a transfer transaction must correspond to a condition in a previous transaction.
+Today, every transaction contains one fulfillment-condition pair. The fulfillment in a transfer transaction must fulfill a condition in a previous transaction.
 
-When a node is asked to check the validity of a transaction, it must do several things, including:
+When a node is asked to check if a transaction is valid, it checks several things. Some things it checks are:
 
-* double-spending checks (for transfer transactions),
-* hash validation (i.e. is the calculated transaction hash equal to its id?), and
-* validation of all fulfillments, including validation of cryptographic signatures if they’re among the conditions.
+* Are all the fulfillments valid? (Do they correctly satisfy the conditions they claim to satisfy?)
+* If it's a creation transaction, is the asset valid?
```

Comment:
The asset being valid usually means that the metadata associated with the asset is valid e.g. a non divisible asset cannot have an amount != 1.

BigchainDB doesn't understand and does not care about the user provided data in the asset itself.